activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/ProcessInstanceIT.java

```diff
@@ -388,6 +388,45 @@ public void resumeShouldPutASuspendedProcessInstanceBackToActiveState() {
         ResponseEntity<CloudProcessInstance> processInstanceEntity = processInstanceRestTemplate.getProcessInstance(startProcessEntity);
         assertThat(processInstanceEntity.getBody().getStatus()).isEqualTo(ProcessInstance.ProcessInstanceStatus.RUNNING);
     }
+    
+    @Test
+    public void adminResumeShouldPutASuspendedProcessInstanceBackToActiveState() {
+        //given
+        ResponseEntity<CloudProcessInstance> startProcessEntity = processInstanceRestTemplate.startProcess(processDefinitionIds.get(SIMPLE_PROCESS));
+        
+        //First suspend process
+        //testadmin should see process instances at admin endpoint
+        keycloakSecurityContextClientRequestInterceptor.setKeycloakTestUser("testadmin");
+        ResponseEntity<Void> responseEntity = adminExecuteRequestSuspendProcess(startProcessEntity);
+        
```

Comment:
@CTI777 it wouldn't hurt to check that the process instance is SUSPENDED at this point.. just to make sure that the next check after resume (RUNNING) make sense

Reply:
Should be implemented